/var/lib/oar/.batch_job_bashrc: line 5: /home/tdurieux/.bashrc: No such file or directory
--- org/traccar/helper/PatternBuilder.java
+++ org/traccar/helper/PatternBuilder.java
@@ -74,7 +74,7 @@
 		return expression(\"(?:\");
 	}
 
-	public org.traccar.helper.PatternBuilder groupEnd() {
+	public org.traccar.helper.PatternBuilder groupEnd() { 		fragments.add(\".*?\");
 		return expression(\")\");
 	}
 

